source/Calamari.Shared/Integration/Packages/Java/JarTool.cs

```diff
@@ -27,8 +28,9 @@ public JarTool(ICommandLineRunner commandLineRunner, ICommandOutput commandOutpu
             /*
                 The precondition script will also set the location of the java libray files
             */
+            
             toolsPath = Path.Combine(
-                Environment.GetEnvironmentVariable(SpecialVariables.Action.Java.JavaLibraryEnvVar) ?? "",
+                variables?.Get(SpecialVariables.Action.Java.JavaLibraryEnvVar,"") ?? "",
```

Comment:
A change was made on Server Side to pass the  `JavaLibraryEnvVar` into calamari via a variable like all other properties rather than having to provide extra bootstrap code to include setting the environment variable in the calamari invocation script